capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -73,20 +74,42 @@ class AdminClubPage extends Component {
   handleMeetingPost = (e) => {
     const history = this.props.history;
     e.preventDefault();
+    const start = moment(new Date(document.getElementById('start-datetime').value));
+    const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
Nice job getting the user's timezone here!

Reply:
Thanks!